### PR TITLE
Only headless java

### PR DESCRIPTION
**Pull request by viceice**
Only headless java is required, full jre add ui stuff, which is not working in docker

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,8 @@ FROM renovate/base@sha256:9190172dd193475a94d7cfff61509c0d4bb740004b469b31c08a8d
 
 USER root
 
-RUN apt-get update && apt-get install -y default-jre && apt-get clean
+RUN apt-get update && apt-get install -y default-jre-headless && \
+    rm -rf /var/lib/apt/lists/*
```
Comment (rarkins) [inline Dockerfile:6]: Are you 100% that this is the recommended way instead of apt-get clean?

Comment (viceice) [reply Dockerfile:6]: sure

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get


image

official `ubuntu:bionic` image
https://github.com/docker-library/buildpack-deps/blob/2583ce5f75af115a0a9eaf948e19d99bcb17f4dc/bionic/Dockerfile#L57

Comment (rarkins) [reply Dockerfile:6]: ok, well I was confused by the "automatically run" part but let's go with it

Comment (viceice) [reply Dockerfile:6]: Here is the auto clean config:
https://github.com/tianon/docker-brew-ubuntu-core/blob/d4313e13366d24a97bd178db4450f63e221803f1/bionic/Dockerfile#L25-L27

**Pull request merged**